app/build.gradle

```diff
@@ -40,6 +40,13 @@ android {
         buildConfigField("String", "CUSTOM_INSTANCE", "\"$CUSTOM_INSTANCE\"")
         buildConfigField("String", "SUPPORT_ACCOUNT_URL", "\"$SUPPORT_ACCOUNT_URL\"")
     }
+    compileOptions {
```

Comment:
this fixed my IDE but idk if we should remove it before the release?

Reply:
we shouldn't even merge it

Reply:
I think the blessed way now is

```groovy
kotlin {
    jvmToolchain(17)
}
```

https://developer.android.com/build/jdks#toolchain